usd/src/usd_parser/USDWorld.cc

```diff
@@ -80,6 +87,25 @@ namespace usd
       std::vector<std::string> primPathTokens =
         ignition::common::split(primPath, "/");
 
+      if (primPathTokens.size() == 1 && !prim.IsA<pxr::UsdGeomCamera>())
```

Comment:
This is making an assumption on the scene graph of the usd, it wouldn't hold if the usd does not come from isaac sim. We should probably note somewhere that the converter only works for isaac sim usds.